src/nock/recorder.ts

```diff
@@ -118,6 +141,87 @@ export async function loadRecording({
   }
 }
 
+type UpdateResult =
```

Comment:
There is a lot of functions in one file, I would divided it to more files and also I would move parts of function into separate functions.

Reply:
good point, I tried to [split it to utils](https://github.com/superfaceai/testing/pull/39/commits/24bae166604386430c7fbf80a5c88b1e2b16e53e)

Reply:
I might split utils more as I'll write tests for it

Reply:
@Jakub-Vacek I wrote tests and was able to test it without further splitting, but let me know if more changes are needed.